experimental/cloudstrapper/playbooks/roles/aws-cleanup/tasks/orc8r-eks.yaml

```diff
@@ -0,0 +1,9 @@
+---
+
+- name: cleaning up orchestator eks cluster
+  aws_eks_cluster:
+    name: "{{ orc8rEksCluster }}"
+    wait: yes
+    state: absent
```

Comment:
That's it ? This cleans up everything :-). Beautiful.

Reply:
:).